test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -680,6 +692,21 @@ private void updateTime(final long leftTime) {
                                         hours, minutes, seconds));
         }
 
+
+        private void pauseToggle() {
+            if (timer.isRunning()) {
+                pauseTimeLeft = endTime - System.currentTimeMillis();
+                timer.stop();
+                label.setText(label.getText() + PAUSED_LABEL_SUFFIX);
+                button.setText(RESUME_BUTTON_LABEL);
+            } else {
+                label.setText(label.getText().replace(PAUSED_LABEL_SUFFIX, ""));
+                endTime = System.currentTimeMillis() + pauseTimeLeft;
```

Comment:
I'd rather call `updateTime(pauseTimeLeft)` here instead of editing the text.

```suggestion
                endTime = System.currentTimeMillis() + pauseTimeLeft;
                updateTime(pauseTimeLeft);
```

Reply:
Yes, this way is better.